phpcs.xml

```diff
@@ -1,9 +1,24 @@
 <?xml version="1.0"?>
 <ruleset name="10up PHPCS">
 	<description>10up PHPCS extended.</description>
+
+	<!-- Scan these directories -->
+	<file>./themes/*</file>
+	<file>./mu-plugins/*</file>
+
+	<!-- Don't scan these directories -->
 	<exclude-pattern>dist/</exclude-pattern>
-	<exclude-pattern>plugins/*</exclude-pattern>
+	<exclude-pattern>./plugins/*</exclude-pattern>
```

Comment:
I feel like this will cause issues pretty soon. To me there are valid reasons why I may need to add custom plugins to the plugins directory and 3rd party plugins to the mu-plugins folder. 

Maybe it makes more sense to not use any wildcards but explicitly specify the folders we want?